src/checkout-buttons/strategies/checkout-button-strategy.ts

```diff
@@ -1,16 +1,16 @@
 import { CheckoutButtonInitializeOptions, CheckoutButtonOptions } from '../checkout-button-options';
 
 export default abstract class CheckoutButtonStrategy {
-    protected _isInitialized = false;
+    protected _isInitialized: {[key: string]: boolean} = {};
```

Comment:
Could you put a space character around the bracket? i.e.: `{ [key: string]: boolean }`